cmd/kube-aggregator/artifacts/hostpath-pods/insecure-etcd-pod.yaml

```diff
@@ -0,0 +1,81 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: kube-aggregator
+  namespace: kube-public
```

Comment:
Would have expected kube-system. Other addons go under kube-systems. Any reason that the aggregator doesn't?